slsa/provenance.go

```diff
@@ -42,6 +42,12 @@ func HostedActionsProvenance(w WorkflowRun) (*intoto.ProvenanceStatement, error)
 		builderID = fmt.Sprintf("https://github.com/%s", t.JobWorkflowRef)
 	}
 
+	buildInvocationID := w.GithubContext.RunID
+	if w.GithubContext.RunAttempt != "" {
+		// NOTE: RunID does not get updated on re-runs so we need to include RunAttempt.
+		buildInvocationID = fmt.Sprintf("%s-%s", w.GithubContext.RunID, w.GithubContext.RunAttempt)
```

Comment:
Could you please add some documentation on the format of this field? 

(Maybe add here https://github.com/slsa-framework/slsa-github-generator#provenance-example and add a note)

Reply:
I will add something related to the invocation ID. I also added an issue to better document the fields determined by the `buildType` where we have made some judgements about the data there. 
https://github.com/slsa-framework/slsa-github-generator/issues/33

Reply:
Done.